tutorials/migrating/breaking_changes_4.1.rst

```diff
@@ -0,0 +1,159 @@
+.. _doc_breaking_changes_4.1:
+
+Breaking changes in Godot 4.1
+=================================
+
+If you are migrating from 4.0 to 4.1, the breaking changes listed here might
+affect you. Changes are grouped by areas/systems.
+
+This article indicates whether each breaking change is *binary compatible* or
+*source compatible*:
+
+- **Binary compatible** - Existing binaries will load and execute successfully without
+  recompilation, and the run-time behavior won't change.
+- **Source compatible** - Source code will compile successfully without changes when
+  upgrading Godot.
+
+Core
+----
+
+========================================================================================================================  =================  ================= ============  ==================
+Change                                                                                                                    Binary Compatible  Source Compatible Introduced    Languages affected
+========================================================================================================================  =================  ================= ============  ==================
+**UndoRedo**
+Method ``create_action`` adds a new ``backward_undo_ops`` optional parameter                                              |❌|               |✔️|              `GH-76688`_   --
+========================================================================================================================  =================  ================= ============  ==================
+
+Animation
+---------
+
+========================================================================================================================  =================  ================= ============  ==================
+Change                                                                                                                    Binary Compatible  Source Compatible Introduced    Languages affected
+========================================================================================================================  =================  ================= ============  ==================
+**AnimationNode**
+Method ``_process`` adds a new ``test_only`` parameter                                                                    |❌|               |❌|              `GH-75759`_   GDScript, C#
+Method ``blend_input`` adds a new ``test_only`` optional parameter                                                        |❌|               |✔️|              `GH-75759`_   --
+Method ``blend_node`` adds a new ``test_only`` optional parameter                                                         |❌|               |✔️|              `GH-75759`_   --
```

Comment:
Nitpick, but the "Languages affected" cell seems to be misaligned by one char.

Reply:
I think you meant the "Introduced" column? Otherwise, I don't see the misalignment.